qunicorn_core/core/pilotmanager/ibm_pilot.py

```diff
@@ -41,6 +41,8 @@
 class IBMPilot(Pilot):
     """The IBM Pilot"""
 
+    qc = None
```

Comment:
better Naming to better represent usage - maybe short comment on why its here (its a fringe case)